src/Apps/Legion/Components/TreeView/TreeView.types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ITreeViewProps {
+    sampleData: Asset[];
+}
+
+export interface Asset {
+    /** Temporary interface for assets. Will be updated */
+
+    /** Display name */
+    name: string;
```

Comment:
Can we make the names more generic? This object can just be a `TreeItem` or something like that. 
And then name can be `text` and the children can just be `children`

Reply:
Made the interface a `Node` since `TreeItem` is a specific thing in the fluent 9 component. Updated the rest as suggested